### ISSUE TITLE
As an inspector, I need the ability to copy a previous report so that I can use it as a template on a job

### ISSUE THREAD

**Issue by AlabamaMike** [firelabinc/nextgen#52]
Need @AriesSoft to confirm the reason why we need this ... are we using a prior report as a template for a report on the job that the inspector is currently completing?

**Comment by AriesSoft** [comment]
Yes, an inspector can copy a report from a previous inspection to save the inspector the time of filling out a report from scratch.  Aries will copy the entire report with the exception of 3 items.  Date, name and signatures, photos.
